src/Artsy/Router/buildServerApp.tsx

```diff
@@ -79,12 +79,12 @@ export function buildServerApp(config: ServerRouterConfig): Promise<Resolve> {
         const headTags = [<style type="text/css">{MediaStyle}</style>]
```

Comment:
```suggestion
        /**
         * An array that gets passed to `react-head`'s provider that will collect the header
         * tags that are rendered by the App. `headTags` is _mutatated_ when it's passed to the App. Beware.
         **/
        const headTags = [<style type="text/css">{MediaStyle}</style>]
```

Reply:
cc @pepopowitz

Reply:
I couldn't accept the change for some reason :(

Applied in a commit.